inc/class_mailhandler.php

```diff
@@ -92,6 +92,23 @@ class MailHandler
 	public $parse_format = 'text';
 
 	/**
+	 * Selects between AdminEmail and ReturnEmail, dependant on if ReturnEmail is filled.
+	 */
+	function from_select()
+	{
+		global $mybb;
+		
+		if($mybb->settings['returnemail'])
```

Comment:
Should be checked via `empty` or at least `trim`, just in case the user entered a space.